src/cookies.js

```diff
@@ -181,8 +183,9 @@ export function getHighestAvailableDomain(win) {
  * @param {time} expirationTime
  * @param {string|undefined} domain
  * @param {!SameSite=} sameSite
```

Comment:
Nit: This signature means that you can either do `foo(sameSite)` or `foo()`. It should be `{!SameSite|undefined}`

Reply:
After reading through [gcc types](https://github.com/google/closure-compiler/wiki/Types-in-the-Closure-Type-System), I'm guessing this means that I should also change `{boolean}` --> `{boolean=}`.

Reply:
No, we'll always call this function with the correct number of arguments, so `=` is incorrect.

Reply:
Sure, I'll switch it back